task5/task5.css

```diff
@@ -0,0 +1,123 @@
+.wrapper {
+    width: 90%;
+    margin: 0 auto;
+}
+
+.content-wrapper {
+    border: 2px solid black;
+    margin-top: 10px;
+}
+
+.content-wrapper .main-header {
+    text-align: center;
+}
+
+.content-wrapper .three-columns {
+    column-count: 3;
+    padding: 10px;
+}
+
+.wrapper .settings {
+    font-weight: bold;
+    margin: 10px;
+    margin-left: 30px;
+}
+
+.three-columns h2:first-of-type {
+    margin-top: 0;
+}
+
+.three-columns p {
+    text-align: justify;
+}
+
+input[type=radio] {
+    display: none;
+}
+
+.sans-serif,
+#sans-serif:checked~.content-wrapper {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.serif,
+#serif:checked~.content-wrapper {
+    font-family: 'Times New Roman', Georgia, Times, serif;
+}
+
+.small-font,
+#small-font:checked~.content-wrapper {
+    font-size: smaller;
+}
+
+.normal-font,
+#normal-font:checked~.content-wrapper {
+    font-size: medium;
+}
+
+.large-font,
+#large-font:checked~.content-wrapper {
+    font-size: larger;
+}
+
+.day-scheme,
+#day-scheme:checked~.content-wrapper {
+    color: black;
+    background-color: white;
+}
+
+.night-scheme,
+#night-scheme:checked~.content-wrapper {
+    color: GHOSTWHITE;
+    background-color: SLATEGRAY;
+}
+
+.sans-serif,
+.serif,
+.small-font,
+.normal-font,
+.large-font,
+.day-scheme,
+.night-scheme {
+    opacity: 0.5;
+}
+
+#sans-serif:checked+.sans-serif,
+#serif:checked+.serif,
+#small-font:checked+.small-font,
+#normal-font:checked+.normal-font,
+#large-font:checked+.large-font,
+#day-scheme:checked+.day-scheme,
+#night-scheme:checked+.night-scheme {
```

Comment:
please set single css class for that to avoid enumeration 7 selectors

Reply:
But I want every pair to change style independently

Reply:
sure, you may add second class

Reply:
this rule is specific not only for sans-serif, serif, small-font etc.
this rule is for selected elements and their label.

Reply:
Ok, I got it now, I underestimated the `+` operator